src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/PingGHEventSubscriber.java

```diff
@@ -59,18 +61,23 @@ protected boolean isApplicable(Item project) {
      */
     @Override
     protected void onEvent(GHEvent event, String payload) {
```

Comment:
That Throwable catch should be the last ditch catch. Propagation to it is not a good pattern.

I'll see if I can decipher what you mean tomorrow on computer not phone. But in general you should catch and log as close and as specific as possible while you know what sense to make...